pipeline/processor.go

```diff
@@ -179,9 +179,9 @@ func (p *processor) processEvent(event *Event) (isSuccess bool, isPassed bool, e
 
 func (p *processor) doActions(event *Event) (isPassed bool) {
 	l := len(p.actions)
-	for index := event.action; index < l; index++ {
+	for index := int(event.action.Load()); index < l; index++ {
```

Comment:
maybe it's possible to avoid conversion?

Reply:
index must be int and atomic can't be just int. I guess we can't avoid conversation.